### PR TITLE
Added CVE-2017-1000414

### PR DESCRIPTION
**Pull request by kurtseifried**

### CODE REVIEW

**Comment by kurtseifried** [comment]
Emailed the requestor, waiting for reply.

**Comment by kurtseifried** [comment]
Added detailed refs, should be good to go now.

File: 2017/1000xxx/CVE-2017-1000414.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  "references": {
+    "reference_data": [
+      {
+        "url": "https://sourceforge.net/p/jpegsnoop/code/HEAD/tree/"
+      },
+      {
+        "url": "https://www.impulseadventure.com/photo/jpeg-snoop-history.html "
+      },
+      {
+        "url": "https://github.com/ImpulseAdventure/JPEGsnoop/search?utf8=%E2%9C%93&q=m_anSofHorzSampFact_Hi&type="
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000414.json]: That url just shows results from a search of the repo for "m_anSofHorzSampFact_Hi". It's not obvious how it relates to the vulnerability.

Would https://github.com/ImpulseAdventure/JPEGsnoop/commit/b4e458612d4294e0cfe01dbf1c0b09a07a8133a4#diff-cf9182aecc9d630e8db2e0e35f1eec65 be a better choice? 

George

File: 2017/1000xxx/CVE-2017-1000414.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+  "data_version": "4.0",
+  "references": {
+    "reference_data": [
+      {
+        "url": "https://sourceforge.net/p/jpegsnoop/code/HEAD/tree/"
+      },
+      {
+        "url": "https://www.impulseadventure.com/photo/jpeg-snoop-history.html "
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000414.json]: The url has a space at the end that causes schema validation to fail.

**Comment by kurtseifried** [comment]
added, thanks

**Comment by cve-team** [comment]
Thanks, Kurt.

Note that I had to manually remove the space from one of the URLs to prevent schema validation failures.

George

**Pull request merged**